jwt/token_service_algorithm.go

```diff
@@ -0,0 +1,102 @@
+package jwt
```

Comment:
I believe we need to rename it now, Becaus it's not a part of token service any more. Something like TokenAlforithm